scanner/src/main/kotlin/scanners/FileCounter.kt

```diff
@@ -41,6 +41,8 @@ object FileCounter : LocalScanner() {
     override val scannerExe = ""
     override val scannerVersion = "1.0"
 
+    override fun bootstrap(): File = File("")
```

Comment:
I believe the better / more generic fix would be to change `LocalScanner` to only bootstrap at all if `scannerExe` is not empty.

Reply:
Removed the commit as you fixed the problem already on `master`.